Cannot mount attack using self-signed TLS/SSL certificate after setting up on GCE

After installing the MiTM on GCE using docs/gce/setup.sh script, attacking using self-signed TLS/SSL certificate fails with:

[ERROR] argument must be an int, or have a fileno() method.
Traceback (most recent call last):
  File "nogotofail/mitm/connection/server.py", line 113, in serve
    cont = conn.bridge(fd)
  File "nogotofail/mitm/connection/connection.py", line 241, in bridge
    return self._bridge_client()
  File "nogotofail/mitm/connection/connection.py", line 329, in _bridge_client
    if self._check_for_ssl(client_request):
  File "nogotofail/mitm/connection/connection.py", line 286, in _check_for_ssl
    return self._handle_hello(client_hello)
  File "nogotofail/mitm/connection/connection.py", line 316, in _handle_hello
    self.connect_ssl(client_hello)
  File "nogotofail/mitm/connection/connection.py", line 158, in connect_ssl
    self._setup_server_connection(server_name)
  File "nogotofail/mitm/connection/connection.py", line 211, in _setup_server_connection
    self._do_ssl_handshake(connection)
  File "nogotofail/mitm/connection/connection.py", line 230, in _do_ssl_handshake
    [connection], [connection], [], remaining)
TypeError: argument must be an int, or have a fileno() method.

--- a/docs/gce/setup_mitm.sh
+++ b/docs/gce/setup_mitm.sh
@@ -9,6 +9,7 @@ LOG_DIR=/var/log/nogotofail
 # Install the relevant source subtree into $INSTALL_DIR
 mkdir -p $INSTALL_DIR $CONFIG_DIR
 cp -a ../../nogotofail $INSTALL_DIR/
+cp ../../dhparam $INSTALL_DIR/
 chown -R root:root $INSTALL_DIR
 chmod -R go-w $INSTALL_DIR
 

